inputsandstrings/js-inputs-and-strings_change-box-appearance/js/index.js

```diff
@@ -16,3 +16,27 @@ For further information check MDN:
 - about CSS declarations with "style": https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style
 
 */
+
+const inputColor = document.querySelector('[data-js="input-color"]');
+const inputRadius = document.querySelector('[data-js="input-radius"]');
+const inputRotation = document.querySelector('[data-js="input-rotation"]');
+const box = document.querySelector('[data-js="box"]');
+
+inputColor.addEventListener("input", function () {
+  const inputColorValue = this.value;
+  box.style.backgroundColor = `hsl(${inputColorValue}deg, 70%, 60%)`;
+});
+
+inputRadius.addEventListener("input", function () {
+  const inputRadiusValue = this.value;
+  if (inputRadiusValue === inputRadius.max) {
+    box.style.borderRadius = "50%";
+  } else {
+    box.style.borderRadius = `${inputRadiusValue}%`;
+  }
```

Comment:
can you explain why you done this if statement?

Reply:
Thanks for your comments! 
My thinking was: if (inputRadiusValue === inputRadius.max) : This condition checks whether the current value of the inputRadiusValue is equal to the maximum value of the inputRadius slider, as specified by inputRadius.max.
When the condition is true (meaning the slider is at its maximum value), the code inside the if block is executed:
box.style.borderRadius = "50%" . It sets the borderRadius property of the box element to "50%" , if it's false the else block is executed.
I not sure if it's necessery, but it works for sure. I will ask tomorrow if i could keep it or not.

Reply:
When it works, then it works. Keep it. 🙂